content/04.study.md

```diff
@@ -79,60 +79,64 @@ Moreover, these nonlinear algorithms can deconvolve the effects of multiple vari
 The ultimate goal is to predict an individual’s tissue-specific, exon-specific splicing patterns from their genome sequence and other measurements to enable a new branch of precision diagnostics that also stratifies patients and suggests targeted therapies to correct splicing defects.
 However, to achieve this we expect that methods to interpret the "black box" of deep neural networks and integrate diverse data sources will be required.
 
-### Transcription factors and RNA-binding proteins
-
-Transcription factors and RNA-binding proteins are key components in gene regulation and higher-level biological processes.
-TFs are regulatory proteins that bind to certain genomic loci and control the rate of mRNA production.
-While high-throughput sequencing techniques such as chromatin immunoprecipitation and massively parallel DNA sequencing (ChIP-seq) have been able to accurately identify targets for TFs, these experiments are both time consuming and expensive.
-Thus, there is a need to computationally predict binding sites and understand binding specificities *de novo* from sequence data.
-In this section we focus on TFs, with the understanding that deep learning methods for TFs are similar to those for RNA-binding proteins, though RNA-specific models do exist [@doi:10.1186/s12859-017-1561-8].
```

Comment:
I understand that RNA binding is a separate problem, but I liked having a sentence that acknowledged this related task and that there are groups working on it.  See my suggestion below.

I'm okay removing `RNA-binding proteins` from the subsection title.